python/source/autowrap/python/boilerplate.d

```diff
@@ -132,21 +132,7 @@ string pydInitMixin(in string libraryName) @safe pure {
 
     if(!__ctfe) return null;
 
-    version(Python2) {
-        return q{
-            import pyd.exception: exception_catcher;
-            import pyd.thread: ensureAttached;
-            import pyd.def: pyd_module_name;
-            extern(C) export void init%s() {
-                exception_catcher(delegate void() {
-                        ensureAttached();
-                        pyd_module_name = "%s";
-                        PydMain();
-                    });
-
-            }
-        }.format(libraryName, libraryName);
-    } else {
+    version(Python_3_0_Or_Later) {
```

Comment:
What's the difference?

Reply:
`Python2` isn't a pyd version, it was set only for the `python27` config of `autowrap:python`.